src/main.tsx

```diff
@@ -1,21 +1,10 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import { createBrowserRouter, RouterProvider } from 'react-router-dom';
-import ErrorPage from './Error';
 import App from './App';
 import './styles/less/petclinic.less';
 
-const router = createBrowserRouter([
```

Comment:
This is the [recommended API](https://reactrouter.com/en/main/start/tutorial#adding-a-router) to write routes.